amplitude_python_sdk/tests/test_chart_annotations_client.py

```diff
@@ -0,0 +1,94 @@
+"""Chart Annotations Client Tests"""
+
+import requests
+from unittest.mock import patch
+
+from datetime import date
+from amplitude_python_sdk.v2.models.charts import ChartAnnotations
+
+from amplitude_python_sdk.v2.clients.chart_annotations_client import ChartAnnotationsAPIClient
+
+_chart_annotations_client_ = ChartAnnotationsAPIClient(api_key='apikey', secret_key='secretkey')
+_annotation_ = ChartAnnotations(app_id=50, date=date(2021, 5, 5), label='labels')
+_annotation_id = '50079'
+_annotation_get_response = {
+    "annotation": {
+        "date": "2021-05-05",
+        "details": "",
+        "id": 50079,
+        "label": "labels"
+    }
+}
+
+_annotation_create_response = {
+    "annotation": {
+        "date": "2021-05-05",
+        "details": "",
+        "id": 50079,
+        "label": "labels"
+    },
+    "success": True
+}
+
+
+@patch('amplitude_python_sdk.common.utils.make_request')
```

Comment:
I'd recommend using the `pytest-mock` library as outlined here: https://changhsinlee.com/pytest-mock/

since the `unittest.mock` library is intended to work with standard `unittest` rather than `pytest`. But the mocks themselves look pretty good!